common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -13,6 +13,7 @@
 import re
 
 from django.conf import settings
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
```

Comment:
@rehan99000 we should avoid importing `site_configuration` to include unnecessary dependency.

Reply:
I agree. @rehan99000 just use `settings.FEATURES.get('PREVIEW_LMS_BASE')`